site/gatsby-site/blog/deepfakes-child-safety/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Deepfakes and Child Safety: A Survey and Analysis of 2023 Incidents and Responses'
 metaTitle: 'Deepfakes and Child Safety: A Survey and Analysis of 2023 Incidents and Responses'
-metaDescription: ""
+metaDescription: "Rising concerns and legal challenges regarding the creation and distribution of AI-generated child sexual abuse materials in 2023."
```

Comment:
I'm concerned about using the meta description as a literal preview. It's intended for summarizing the page, not being the content exactly.

What do you think about adding and using instead a specific dedicated `previewText`? That might also simplify the logic: use `previewText` first before falling back to other content.

I understand this could be a pain with supporting both Prismic and MDX. We can try to simplify too if that's the case.

Lastly, a working preview would be really helpful here, but I see there's problems with the workflow to deploy. Let me know if you're blocked here.

Reply:
`previewText` sounds good. Not a problem for prismic because we only use rich text there so the preview text is already markdown and html free. For mdx we can add the `previewText` and if doesn't exist falleback to the content, removing markdowns.